packages/replay-canvas/.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+// Note: All paths are relative to the directory in which eslint is being run, rather than the directory where this file
+// lives
+
+// ESLint config docs: https://eslint.org/docs/user-guide/configuring/
+
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+  overrides: [
+    {
+      files: ['src/**/*.ts'],
+      rules: {
+        '@sentry-internal/sdk/no-unsupported-es6-methods': 'off',
+      },
+    },
+    {
+      files: ['jest.setup.ts', 'jest.config.ts'],
+      parserOptions: {
+        project: ['tsconfig.test.json'],
+      },
+      rules: {
+        'no-console': 'off',
+      },
+    },
+    {
+      files: ['test/**/*.ts'],
+
+      rules: {
+        // most of these errors come from `new Promise(process.nextTick)`
```

Comment:
l: we can probably remove all of these overrides here, as they probably do not apply/are necessary for this package?